depends/packages/rapidcheck.mk

```diff
@@ -0,0 +1,18 @@
+package=rapidcheck
+$(package)_version=10fc0cb
+$(package)_download_path=https://github.com/MarcoFalke/rapidcheck/archive
+$(package)_file_name=$(package)-$($(package)_version).tar.gz
+$(package)_sha256_hash=9640926223c00af45bce4c7df8b756b5458a89b2ba74cfe3e404467f13ce26df
+
+define $(package)_config_cmds
+  cmake -DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=true .
+endef
+
+define $(package)_build_cmds
+  $(MAKE) && \
+  mkdir -p $($(package)_staging_dir)$(host_prefix)/include && \
```

Comment:
indeed, this is necessary because rapidcheck has no `make install`

Reply:
This is an open PR on the repo

https://github.com/emil-e/rapidcheck/pull/152

Reply:
Now that the PR is merged, should this be switched over?